core/input/input_map.cpp

```diff
@@ -130,23 +129,21 @@ void InputMap::erase_action(const StringName &p_action) {
 
 TypedArray<StringName> InputMap::_get_actions() {
 	TypedArray<StringName> ret;
-	List<StringName> actions = get_actions();
-	if (actions.is_empty()) {
-		return ret;
-	}
+	LocalVector<StringName> actions = get_actions();
+
+	ret.resize(actions.size());
 
-	for (const StringName &E : actions) {
-		ret.push_back(E);
+	for (unsigned i = 0; i < actions.size(); i++) {
+		ret[i] = actions[i];
 	}
 
 	return ret;
 }
 
-List<StringName> InputMap::get_actions() const {
-	List<StringName> actions = List<StringName>();
-	if (input_map.is_empty()) {
-		return actions;
-	}
+LocalVector<StringName> InputMap::get_actions() const {
+	LocalVector<StringName> actions = LocalVector<StringName>();
```

Comment:
`LocalVector<StringName> actions;`

Reply:
You can use suggestions (the little page with `+` and `-`) to make specific suggestions for changes, this is far more helpful, like so:
```suggestion
	LocalVector<StringName> actions;
```

Reply:
Thank you for the tip, I will try that next time.